docs/types/value-types.rst

```diff
@@ -161,18 +161,33 @@ Fixed Point Numbers
     cannot be assigned to or from.
 
 ``fixed`` / ``ufixed``: Signed and unsigned fixed point number of various sizes. Keywords ``ufixedMxN`` and ``fixedMxN``, where ``M`` represents the number of bits taken by
-the type and ``N`` represents how many decimal points are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
+the type and ``N`` represents how many fractional decimal digits are available. ``M`` must be divisible by 8 and goes from 8 to 256 bits. ``N`` must be between 0 and 80, inclusive.
 ``ufixed`` and ``fixed`` are aliases for ``ufixed128x18`` and ``fixed128x18``, respectively.
 
+Fixed point types are used to represent fractional numbers with a fixed precision. The number ``1.5`` is stored in a ``fixed8x2`` (8 bit number with
+two fractional digits) as ``1.50``. In the EVM, it is stored in exactly the same way as the number ``150`` would be stored in an ``int8``.
+The largest and smallest numbers representable in a ``fixed8x2`` are ``1.27`` and ``-1.28``, respectively.
+
+Type conversions:
+
+Implicit conversions from integer and fixed point types to ``fixedMxN`` types can be performed,
+as long as all values can still be represented ("widening" conversion).
+
+Explicit conversions to integer types truncate the fractional part (towards zero) and can only be performed if the integer
```

Comment:
How do you imagine this conversion to happen? Do we want this already or should we just let people manually truncate if they need to? (As they mostly do with scaling between various precisions.)

Reply:
We can leave it out, but if we only allow conversion to bytes type, it might be a bit cumbersome to use. How do you imagine the manual truncation to work?